WindowsMRAssetConverter/CommandLine.cpp

```diff
@@ -44,6 +46,7 @@ void CommandLine::PrintHelp()
         << indent << "[" << std::wstring(PARAM_LOD) << " <path to each lower LOD asset in descending order of quality>]" << std::endl
         << indent << "[" << std::wstring(PARAM_SCREENCOVERAGE) << " <LOD screen coverage values>]" << std::endl
         << indent << "[" << std::wstring(PARAM_MAXTEXTURESIZE) << " <Max texture size in pixels, defaults to 512>]" << std::endl
+		<< indent << "[" << std::wstring(PARAM_GENTANGENTS) << " <Generate tangent space - requires UV coordinates>]" << std::endl

```

Comment:
The text in <> should be the parameter format. In this case not needed since -gen-tangents doesn't require additional parameters